app/models/session.rb

```diff
@@ -32,9 +32,24 @@ class Session < ApplicationRecord
 
   validates_datetime :ends_at, after: :starts_at
 
-  scope :starts_at, ->(date) { where(starts_at: date.all_day) }
+  scope :starts_at, ->(date) { where("date(starts_at) = ?", date) }
+  scope :past, -> { where(ends_at: ...Time.current) }
+  scope :live, -> { where("? BETWEEN starts_at AND ends_at", Time.current) }
+  scope :starting_soon, -> { where("starts_at BETWEEN ? and ?", Time.current, 1.hour.from_now) }
 
   def self.ransackable_attributes(_auth_object = nil)
     %w[title]
   end
+
+  def live?
+    Time.current.between?(starts_at, ends_at)
+  end
+
+  def starting_soon?
```

Comment:
Doesn't seem to be correct. I created a session that `starts_at` on Aug 13 and ends on Aug14 and it is being marked as "starting soon"
![image](https://github.com/user-attachments/assets/4a36f9cb-33fe-430b-a81f-810ce0afa08b)